rllib/execution/rollout_ops.py

```diff
@@ -77,35 +80,51 @@ def synchronous_parallel_sample(
         # No remote workers in the set -> Use local worker for collecting
         # samples.
         if worker_set.num_remote_workers() <= 0:
-            sample_batches = [worker_set.local_worker().sample()]
+            sample_batches_or_eps = [worker_set.local_worker().sample()]
         # Loop over remote workers' `sample()` method in parallel.
         else:
-            sample_batches = worker_set.foreach_worker(
+            sample_batches_or_eps = worker_set.foreach_worker(
                 lambda w: w.sample(), local_worker=False, healthy_only=True
             )
             if worker_set.num_healthy_remote_workers() <= 0:
                 # There is no point staying in this loop, since we will not be able to
                 # get any new samples if we don't have any healthy remote workers left.
                 break
         # Update our counters for the stopping criterion of the while loop.
-        for b in sample_batches:
+        for b_or_e in sample_batches_or_eps:
             if max_agent_steps:
-                agent_or_env_steps += b.agent_steps()
+                # TODO (simon): Add `get_agent_steps` to MAE.
+                # TODO (sven): Can't we update here the main counters, too?
+                agent_or_env_steps += (
+                    b_or_e.agent_steps()
+                    if isinstance(b_or_e, (SampleBatch, MultiAgentBatch))
```

Comment:
see above: Let's unify these APIs here:
MultiAgentEpisode should have both methods, too: `agent_steps()` and `env_steps()`.
Just like MultiAgentBatch. For our own sanity and to avoid these if blocks.

Reply:
I at least use now the new `agent_steps()` and `env_steps()` methods of the `MultiAgentEpisode`